sources/packages/forms/src/form-definitions/educationprogram.json

```diff
@@ -1691,12 +1693,12 @@
                     "tags": [],
                     "properties": {},
                     "conditional": {
-                        "show": true,
-                        "when": "programDeliveryTypes",
-                        "eq": "deliveredOnline",
+                        "show": "",
+                        "when": null,
+                        "eq": "",
                         "json": ""
                     },
-                    "customConditional": "",
+                    "customConditional": "const isBCInstitution = data.isBCPublic || data.isBCPrivate;\nshow = (isBCInstitution === false && data.programDeliveryTypes.deliveredOnline);",
```

Comment:
@lewischen-aot I see the reason why dry run submission is failing. The following DTO EducationProgramAPIInDTO is not allowing the variables `isBCPublic` and `isBCPrivate` from form submission to the API payload.

Reply:
Thanks @dheepak-aot. You're absolutely right 👍 I've made that change in the API side to retrieve the values thanks to @andrepestana-aot and @andrewsignori-aot.